### PR TITLE
Create PowerSession.json

### PR DESCRIPTION
**Pull request by ibigbug**
Add [PowerSession](https://github.com/ibigbug/PowerSession) to bucket

### CODE REVIEW

File: bucket/PowerSession.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    "hash": "3b07c318dc4a5e1d25f5d086d706a1eb1ef074b1ef60fb498dd7954de6b09068",
+    "bin": "PowerSession.exe",
+    "checkver": {
+        "github": "https://github.com/ibigbug/PowerSession",
+        "regex": "tree/release-v([\\d.]+)"
+    },
+    "autoupdate": {
+        "url": "https://github.com/ibigbug/PowerSession/releases/download/release-v$version/PowerSession.exe"
+    }
+}
```
[the remark this answers is not in the document]
Comment (ibigbug) [reply bucket/PowerSession.json]: Sorry didn't quite get this here, could u please add more details what you suggesting?

[the remark this answers is not in the document]
Comment (Ash258) [reply bucket/PowerSession.json]: Files have to end with empty new line

**Comment by r15ch13** [comment]
Cool! Could you provide a smaller build? ~60mb is a bit much. Or does it require a really new .NET version?

Also, would be nice if it would work with `pwsh.exe` and `cmd.exe` as well.
(I just modified `RecordCommand.cs` and it works :grin:)

**Comment by ibigbug** [comment]
@r15ch13 yeah I'll try to reduce the binary size. found several articles talking about it.

Please checkout the latest version, supports custom commands :)

**Comment by Ash258** [comment]
Migrate this PR to https://github.com/ScoopInstaller/Main bucket

**Pull request closed without merging**